modules/graphstudio/pages/map-data-to-graph.adoc

```diff
@@ -192,7 +199,8 @@ When selected, the dashed line becomes orange (active), and the right working pa
 
 image::1.png[]
 
-To map a column in the data file to a vertex or edge field, first select the row representing the data column in the left table, then click the target field in the right table.
+To map a column in the data file to a vertex or edge field, drag and drop a row (representing a table in a CSV or a key in a JSON file) from the data file to a target on a vertex or edge.
```

Comment:
This section is not really accurate. 
A row here does not represet a table in a csv, it represents a column.
Users also won't be drag and dropping to a vertex or edge. They will be drag and dropping from a "row" to the "attribute/primary id/source or target id" of a vertex or edge.
This part is in the right chart on the map data to graph page. Users map from a